_posts/2019-01-30-gcal-for-project-management.md

```diff
@@ -0,0 +1,79 @@
+
+2. **Ombu Labs Products & Open Source**. In this calendar, I have our team's weekly [OmbuShop](http://www.ombushop.com/) call, [FastRuby.io](https://fastruby.io/) [pair programming](https://www.ombulabs.com/blog/agile/pair-programming/joys-and-woes-of-pair-programming.html) sessions and some hours that are blocked off to work on open source projects.
+
+3. **Client Projects**. This calendar contains all events relating to our client projects. This includes demo calls, check-ins with clients and pair programming sessions.
+
+<img src="/blog/assets/images/multiple-calendars.png" alt="Multiple Calendars">
+
+#### Shared Calendars
+
+Shared calendars are a great way to get your team on board with a group of scheduled events. At [Ombu Labs](https://www.ombulabs.com), we use the shared calendars feature in our **Vacations Calendar** and our **Birthdays Calendar**.
+
+When a team member is going on vacation, they mark the days they will be away on the shared **Vacations Calendar**, so that their absence appears on the top of everyone else's calendar. This is preferable to the [_*Out of Office*_](https://gsuiteupdates.googleblog.com/2018/06/better-manage-your-work-and-personal_27.html ) feature, which only shows up on your individual calendar.
+
+The shared **Birthday Calendar** works similarly. If the calendar is enabled, team member birthdays will appear at the top of your calendar. Both of these shared calendars can be hidden by each team member to reduce clutter as well.
+
+<img src="/blog/assets/images/shared-calendar.png" alt="Shared Calendars">
+
+#### Managing Multiple Clients
+
+At [Ombu Labs](https://www.ombulabs.com), we typically work on two to three client projects at a time. As an Account Manager, it is important for me to dedicate time to working and managing each one of them.
+
+**Color Coding**. In my calendar, I like to color-code my calls and meetings by client. If the team has two weekly calls with a client, both of those calls in my calendar will be the color purple, for example. This helps to get an idea of how much time in a week is spent on each client. If my weekly calendar has no purple, it tells me that I should schedule a check-in with the client or the team working on the project.
+
+**Schedule Consistency**. Another way to keep consistency with client projects is to schedule project calls around the same time of day or the same day of the week. For a client with once a week check-in calls, it could be helpful to schedule that check-in every Monday morning.
+
+For a client that requests a weekly demo call and a weekly status call, it could be helpful to have both calls, for example, in the afternoon, say 4pm. Obviously this is not always possible, but it can be helpful for the client and the team to get into a routine of having work ready to go at the same time of day or day of the week.
```

Comment:
You could mention that we prefer to have a demo call every two weeks and a planning call every two weeks. So, one week we would have a planning call to talk priorities and new stories, the next week we would have a demo call, and so on...